packages/openneuro-components/package.json

```diff
@@ -0,0 +1,126 @@
+    "react-json-tree": "0.15.0",
+    "react-markdown": "5.0.3",
+    "react-router": "^5.1.2",
+    "react-router-dom": "^5.1.2",
+    "react-spring": "^8.0.27",
+    "react-toastify": "6.0.9",
+    "react-virtualized": "remorses/react-virtualized-fixed-import.git#9.22.3",
+    "resize-observer-polyfill": "^1.5.1",
+    "semver": "^5.5.0",
+    "serviceworker-webpack-plugin": "^1.0.1",
+    "subscriptions-transport-ws": "^0.9.9",
+    "superagent": "^3.6.0",
+    "universal-cookie": "^4.0.4",
+    "universal-cookie-express": "^4.0.3",
+    "url-search-params-polyfill": "^2.0.1",
+    "urlite": "^1.2.8",
+    "xmldoc": "^1.1.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.13.15",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/plugin-proposal-object-rest-spread": "^7.6.2",
+    "@babel/plugin-proposal-optional-chaining": "^7.6.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.6.2",
+    "@babel/preset-env": "^7.6.3",
+    "@babel/preset-react": "^7.6.3",
+    "@babel/preset-typescript": "^7.7.4",
+    "@babel/runtime": "^7.6.3",
+    "@babel/runtime-corejs3": "^7.6.3",
+    "@loadable/babel-plugin": "^5.10.3",
+    "@mdx-js/react": "^1.6.22",
+    "@storybook/addon-a11y": "^6.2.8",
+    "@storybook/addon-actions": "^6.2.8",
+    "@storybook/addon-docs": "^6.2.8",
+    "@storybook/addon-essentials": "^6.2.8",
+    "@storybook/addon-links": "^6.2.8",
+    "@storybook/addons": "^6.2.8",
+    "@storybook/preset-scss": "^1.0.3",
+    "@storybook/react": "^6.2.8",
+    "@storybook/theming": "^6.2.8",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@types/babel__core": "^7",
+    "@types/mdx-js__react": "^1",
+    "@types/node": "^14.14.37",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^24.9.0",
+    "babel-loader": "^8.2.2",
+    "cache-loader": "^1.2.2",
+    "clean-webpack-plugin": "^0.1.19",
+    "compression-webpack-plugin": "^1.0.0",
+    "connect-history-api-fallback": "^1.5.0",
+    "copy-webpack-plugin": "^4.5.2",
+    "css-loader": "^5.2.1",
+    "enzyme": "3.9.0",
+    "esbuild-plugin-globals": "^0.1.1",
+    "express": "^4.17.1",
+    "file-loader": "^1.1.11",
+    "jest": "^26.6.3",
+    "jest-fetch-mock": "^1.4.1",
+    "live-server": "^1.2.1",
+    "mini-css-extract-plugin": "^0.4.1",
+    "moment-timezone": "^0.5.31",
+    "node-sass": "4.14.1",
+    "object.fromentries": "^2.0.0",
+    "react-addons-test-utils": "^15.6.2",
+    "react-test-renderer": "^16.8.6",
+    "rollup-plugin-polyfill-node": "^0.6.2",
+    "sass": "^1.32.8",
+    "sass-loader": "^10.1.1",
+    "style-loader": "^2.0.0",
+    "ts-node-dev": "^1.1.6",
+    "url-loader": "^1.0.1",
+    "vite": "2.1.5",
+    "webpack": "^4.20.2",
+    "webpack-cli": "^3.1.2",
+    "webpack-dev-server": "^3.1.9",
+    "webpack-merge": "^4.1.4",
+    "webpack-visualizer-plugin": "^0.1.11",
+    "worker-plugin": "^3.2.0"
```

Comment:
This looks like it's just a copy of the app deps? I think we should try to reduce this to packages used in this one.